scripts/docker/zeppelin-quick-start/README.md

```diff
@@ -0,0 +1,29 @@
+# Apache Zeppelin Quickstart with Docker Compose
+
+## Getting Started
+
+### Install Docker
+Please go to [install](https://www.docker.com/) to install Docker.
+
+### Zeppelin Only
+#### Run docker-compose
+```bash
+docker-compose -f docker-compose-zeppelin-only.yml up
```

Comment:
`docker-compose` was deprecated for a long time and removed in the recent Docker Desktop versions, use v2 `docker compose` instead.

Reply:
@pan3793 I have made the changes.